libgadget/treewalk.c

```diff
@@ -931,7 +920,7 @@ cull_node(const TreeWalkQueryBase * const I, const TreeWalkNgbIterBase * const i
  * max(P[other].Hsml, iter->Hsml).
  *
  * Particle that intersects with other domains are marked for export.
- * The hosting nodes are exported as well, if tw->UseNodeList is True.
+ * The hosting nodes are exported as well.
```

Comment:
Are exported nodes always the leaves of the global tree?

Reply:
Yes, I think so.